test/integ-new/main_new_test.go

```diff
@@ -180,6 +182,39 @@ func TestAPIAndPublicKeyService(t *testing.T) {
 	}
 }
 
+func TestUIServiceHealthCheck(t *testing.T) {
+	t.Parallel()
+
+	addr := cfg.UIServiceAddr
+	wantStatusCode := http.StatusOK
+	endpoint := "/health"
+
+	ctx := context.Background()
+
+	client := &http.Client{
+		Timeout: 5 * time.Second,
+	}
+
+	uri := fmt.Sprintf("%s%s", addr, endpoint)
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, uri, nil)
+	if err != nil {
+		t.Fatalf("failed to create request: %v", err)
+	}
+
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", cfg.UIServiceIDToken))
+
+	resp, err := client.Do(req)
+	if err != nil {
+		t.Fatalf("client failed to get response: %V", err)
+	}
+
+	defer resp.Body.Close()
+
+	if want, got := wantStatusCode, resp.StatusCode; got != want {
+		t.Errorf("response status code got unexpected diff (-want, +got):\n - %v\n+ %v\n", want, got)
```

Comment:
nit: should we include the response body to add more info when test fails, see example https://github.com/abcxyz/jvs/blob/9b2bca2472d199a91e9802be7c4f73fdac3c69ea/pkg/cli/ui_server_test.go#L129

Reply:
Updated! Thanks!